app/views/devise/invitations/new.html.erb

```diff
@@ -0,0 +1,11 @@
+<h2><%= t "devise.invitations.new.header" %></h2>
+
+<%= simple_form_for resource, as: resource_name, url: invitation_path(resource_name), html: { method: :post} do |f| %>
+  <%= devise_error_messages! %>
+
+<% resource.class.invite_key_fields.each do |field| -%>
+  <%= f.input field %>
+<% end -%>
+
```

Comment:
I noticed this still renders although I did not test if it actually works.  I would say we have two options here- either disable it completely or lock it down to admins only.  That being said, your form that takes multiple email addresses is definitely nicer.